Form-Controls/index.html

```diff
@@ -1,28 +1,70 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="utf-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <title>My form exercise</title>
-  <meta name="description" content="">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>T-shirt Order Form</title>
   <link rel="stylesheet" href="styles.css">
 </head>
 <body>
-  <header>
-    <h1>Product Pick</h1>
-  </header>
-  <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
-
-  </main>
+  <div class="form-container">
+    <h2>T-shirt Order Form</h2>
+    <form name="tshirtForm">
+      <label for="name">Customer's Name:</label>
+      <input type="text" id="name" name="name" required minlength="2" pattern="[A-Za-z]+">
+      <div class="error-message">Please enter a valid name </div>
+
+      <label for="email">Customer's Email:</label>
+      <input type="email" id="email" name="email" required>
+      <div class="error-message">Please enter a valid email address.</div>
```

Comment:
What will happen with these error messages? Can you talk through what you expect?